README.md

```diff
@@ -10,6 +10,7 @@
 - Style them using CSS
```

Comment:
If check the [Microverse template](https://github.com/microverseinc/readme-template) you will see there is a section where you're supposed to add `Live Demo Link`.

Kindly add it to your README file.